README.md

```diff
@@ -14,6 +14,18 @@ result = elevated_runner.powershell_elevated('dir', 'Administrator', 'password')
 puts "Std out: #{result.output}"
 ```
 
+### Impersonating a service account
+By passing a `nil` password, winrm-elevated will assume that the command should run as a service account:
```

Comment:
Thats actually the term MS uses in [the api](https://msdn.microsoft.com/en-us/library/windows/desktop/aa382577(v=vs.85).aspx)`TASK_LOGON_SERVICE_ACCOUNT`: `Indicates that a Local System, Local Service, or Network Service account is being used as a security context to run the task.` I used System because thats what I needed to test.